From 800647b2153a9755835f54b500d15db34d7e69fd Mon Sep 17 00:00:00 2001
From: Xiaolin Ju <Ju.xl@ntu.edu.cn>
Date: Fri, 30 Jun 2017 14:32:29 +0800
Subject: [PATCH 2/2] maindmap.c @ Mindmap Branch

---
 mindmap.c | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 mindmap.c

diff --git a/mindmap.c b/mindmap.c
new file mode 100644
index 0000000..35ece19
--- /dev/null
+++ b/mindmap.c
@@ -0,0 +1 @@
+## mind map .c
-- 
2.11.0 (Apple Git-81)

